guides/source/testing.md

```diff
@@ -701,9 +701,6 @@ raised. Usually this indicates that your schema is not fully migrated. Running
 the migrations against the development database (`bin/rails db:migrate`) will
 bring the schema up to date.
 
-NOTE: If there were modifications to existing migrations, the test database needs to
-be rebuilt. This can be done by executing `bin/rails db:test:prepare`.
```

Comment:
We should tell them to use `bin/rails test:db` instead, which is a publicly documented task